### PR TITLE
Adds a warning to the classic page to direct users to the news

### PR DESCRIPTION
**Pull request by dakota002**
Is something like this what you had in mind?

image

### CODE REVIEW

**Review by lpsinger — CHANGES_REQUESTED** [review]
- [ ] Use the [react-uswds component](https://trussworks.github.io/react-uswds/?path=/story/components-alert--success).
- [ ] Explain _why_ the page will be updated.

**Comment by dakota002** [comment]
@lpsinger Is this looking good, or you thinking more verbose?
image

**Comment by lpsinger** [comment]
> @lpsinger Is this looking good, or you thinking more verbose?

Yes, that looks good.

File: app/routes/circulars/classic.mdx [hunk 1]
```diff
@@ -11,6 +11,15 @@ import {
...
 import { validSubjectKeywords } from './circulars.lib'
 
+<div className="usa-alert usa-alert--warning">
+  <div className="usa-alert__body">
+    <p className="usa-alert__text">
+      This page will be updated on April 17, 2023. See [news and
+      announcements](/news) for more information.
+    </p>
+  </div>
+</div>
```
Comment (lpsinger) [inline app/routes/circulars/classic.mdx:21]: Use the react-uswds Alert component here.

File: app/routes/circulars/classic.mdx [hunk 2]
```diff
@@ -36,11 +45,11 @@ Circulars allow the GRB community to **email** messages to a central queue where
 
 To receive or submit to this GCN Circular list, you must first be added to the vetted list. The purpose is to prevent spam and other unwanted messages. Incoming emails from an address that is not part of the list will be rejected.
 
-You can request access through the [sign up page](https://gcn.gsfc.nasa.gov/gcn_circ_signup.html). There are no restrictions to being added to the list. It is recommended that you send an email to [Scott Barthelmy](mailto:scott.d.barthelmy@nasa.gov) who will confirm that your account name and domain are correct.
+You can request access through the [sign up page](https://gcn.gsfc.nasa.gov/gcn_circ_signup.html). There are no restrictions to being added to the list. It is recommended that you contact the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn).
```
Comment (lpsinger) [inline app/routes/circulars/classic.mdx:48]:
```suggestion
You can request access through the [sign up page](https://gcn.gsfc.nasa.gov/gcn_circ_signup.html). There are no restrictions to being added to the list. It is recommended that you contact the [GCN Team](/contact?service=gcn-classic).
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/circulars/classic.mdx (hunk 1 above)
Comment (dakota002) [reply app/routes/circulars/classic.mdx]: Updated in latest commit

**Review by lpsinger — APPROVED** [review]
Please don't merge yet. I'm doing some maintenance on the CD stuff.

**Pull request merged**